src/Ginger/Id.elm

```diff
@@ -6,7 +6,14 @@ module Ginger.Id exposing
     , fromJson
     )
 
-{-|
+{-| The Ginger resource id. We 'tag' the resource id with type `Int`
+using [elm-tagged](https://package.elm-lang.org/packages/joneshf/elm-tagged/2.1.1/).
```

Comment:
Thanks for the link - educating read and nice solution.

Reply:
Thanks for the typo fixes!